projects/rabbitmq.com/package.yml

```diff
@@ -0,0 +1,52 @@
+distributable:
+  url: https://github.com/rabbitmq/rabbitmq-server/releases/download/v{{version}}/rabbitmq-server-generic-unix-{{version}}.tar.xz
+  strip-components: 1
+versions:
+  github: rabbitmq/rabbitmq-server
+warnings:
+  - vendored
+dependencies:
+  erlang.org: '*'
+build:
+  dependencies:
+    python.org: ~3.11
+  script:
+    - run: |
+        cp -r $SRCROOT/* ./
+        rm -r *.pkgx.*
+        rm pkgx.yaml
+      working-directory: "{{prefix}}"
+    - run: mkdir -p lib/rabbitmq og/rabbitmq
+      working-directory: "{{prefix}}/var"
+    - run: |
+        ln -s ../sbin/rabbitmqctl rabbitmqctl || true
+        ln -s ../sbin/rabbitmq-defaults rabbitmq-defaults || true
+        ln -s ../sbin/rabbitmq-diagnostics rabbitmq-diagnostics || true
+        ln -s ../sbin/rabbitmq-env rabbitmq-env || true
+        ln -s ../sbin/rabbitmq-plugins rabbitmq-plugins || true
+        ln -s ../sbin/rabbitmq-queues rabbitmq-queues || true
+        ln -s ../sbin/rabbitmq-server rabbitmq-server || true
+        ln -s ../sbin/rabbitmq-streams rabbitmq-streams || true
+        ln -s ../sbin/rabbitmq-upgrade rabbitmq-upgrade || true
+        ln -s ../sbin/vmware-rabbitmq vmware-rabbitmq || true
+      working-directory: "{{prefix}}/bin"
+provides:
+  - bin/rabbitmqctl
+  - bin/rabbitmq-defaults
+  - bin/rabbitmq-diagnostics
+  - bin/rabbitmq-env
+  - bin/rabbitmq-plugins
+  - bin/rabbitmq-queues
+  - bin/rabbitmq-server
+  - bin/rabbitmq-streams
+  - bin/rabbitmq-upgrade
+  - bin/vmware-rabbitmq
```

Comment:
Should be fine to leave everything alone in sbin

Reply:
Oh, I thought that `provides` must be in `bin/` dir

Reply:
We have others with sbins. Unbound comes to mind.